packages/test/service-load-test/src/runner.ts

```diff
@@ -92,6 +94,51 @@ async function runnerProcess(
     }
 }
 
+function scheduleFaultInjection(
+    ds: FaultInjectionDocumentServiceFactory,
+    container: Container,
+    runConfig: IRunConfig) {
+    const schedule = ()=>{
+        const leaveTime = runConfig.testConfig.readWriteCycleMs * 5 * Math.random();
+        printStatus(runConfig.runId, `fault injection in ${(leaveTime / 60000).toString().substring(0,4)} min`);
+        setTimeout(() => {
+            if(container.connected && container.resolvedUrl !== undefined) {
```

Comment:
Is the check for `container.connected` redundant with the check for `deltaConn !== undefined` below?

Reply:
no. see the assert in `injectError` if we are not conencted the error handler might not be configured